### ISSUE TITLE
html content messed up for 2 anchor tags

### ISSUE THREAD

**Issue by santosharun** [remarkjs/react-markdown#322]
`const markdown = "two anchors (<a href="www.google.com">one</a>, <a href="www.google.com">two</a>)"`

```
<ReactMarkdown
          source={markdown}
          escapeHtml={false}
          linkTarget='_blank'
/>
```

gives the output as `<p>two anchors (<span><a href="www.google.com"></a></span>one<a>, <span><a href="www.google.com"></a></span>two</a>)</p>`

The above code places the text outside of the anchor tag. Also `linkTarget` doesn't seem to work properly. I am using version 4.0.6

**Comment by Manc** [comment]
HTML tags always seem to be falsely wrapped into `span` tags. None of the test cases seem to test for "custom" HTML in markdown, so I came to the conclusion this package/component not support HTML tags in markdown (yet). In my case (because I needed only a quick solution to try something out), I used NPM package `markdown-to-jsx` instead.

**Comment by bjax13** [comment]
I am getting the same issue.

**Comment by tr1s** [comment]
Same issue here.

**Comment by T04435** [comment]
Any updates on this

**Comment by rwlaschin** [comment]
Any update?

**Comment by ivan-kleshnin** [comment]
> HTML tags always seem to be falsely wrapped into span tags.

Tags that "look" inline are wrapped in `span`. Tags that "look" block (take multiple strings) are wrapped in `div`.  🤷

**Comment by ChristianMurphy** [comment]
this will be resolved in #428

**Comment by wooorm** [comment]
I believe this is fixed on 5.0.0, [with this change](https://github.com/remarkjs/react-markdown/commit/22a5e49ae9dde84257d043f033d1b640b78c7b34#diff-e48a0b02aa7a8f65a453b849e7345933ee0072fb6d0c2a109c83f43cc8fae8c6). Could someone confirm whether that works? Otherwise it’s just the HTML parser being funky and the plan is for that to be replaced next major.

**Comment by wooorm** [comment]
Btw, this is a duplicate of https://github.com/remarkjs/react-markdown/issues/320, so I’ll close this, and we can discuss in that issue
